script/configs/custom_analysis.yaml

```diff
@@ -6,6 +6,11 @@
 # All entries here shoud include a comment explaining why they use different
 # options.
 
+# DO NOT move or delete this file without updating
+# https://github.com/dart-lang/sdk/blob/master/tools/bots/flutter/analyze_flutter_plugins.sh
+# which references this file from source, but out-of-repo.
+# Contact stuartmorgan or devoncarew for assistance if necessary.
```

Comment:
Has this been updated, or will it be?

Reply:
I'll update it post-merge; in the short term it'll keep working since the repo will still be there.